src/stylesheets/dsdl/core.css

```diff
@@ -10,15 +10,6 @@ body {
   line-height: var(--line-height-normal);
 }
 
-h1, .h1,
-h2, .h2,
-h3, .h3,
-h4, .h4,
-h5, .h5,
-h6, .h6 {
-  font-family: var(--dsdl-heading-font-stack);
-}
-
```

Comment:
I've been learning a lot from following these PRs 😄 

Now this removal makes sense to me, since we have the same definitions below 👍 I was just thinking that maybe we can also move the `h5` block below so that it follows the order 6, 5, 4, etc. Not a big deal but just noticed that.